squarelet/oidc/viewsets.py

```diff
@@ -19,7 +19,7 @@ class ClientViewSet(viewsets.ModelViewSet):
 
     queryset = Client.objects.none()
     serializer_class = ClientSerializer
-    permission_classes = (DjangoObjectPermissions,)
+    permission_classes = (IsAdminUser,)
```

Comment:
This should be left as `DjangoObjectPermissions` and the permissions should be controlled through the `squarelet/oidc/rules.py` file